Sources/GRPC/ClientConnection.swift

```diff
@@ -265,7 +296,7 @@ extension ClientConnection {
   /// - Parameter backoffIterator: An `Iterator` for `ConnectionBackoff` providing a sequence of
   ///     connection timeouts and backoff to use when attempting to create a connection.
   private class func makeChannel(
-    configuration: Configuration,
+    configuration: ClientConnection.Configuration,
```

Comment:
Is the `ClientConnection.` prefix necessary?

Reply:
Nope, will remove them, that was WIP that didn't get reverted.